### DISCUSSION TITLE
Delete Privately hosted branch no longer a free feature

### DISCUSSION THREAD

**Discussion by jq001p** [gitkraken/vscode-gitlens#3556] [Q&A] [answered]
Is it no longer possible to delete branches with the free version of gitlens? Every time I try and delete one I am asked to start a pro trail or get a pro account which I used to be able to do less then a week ago with no issue.

**Comment by jkelroy** [comment, marked as answer]
Hello @jq001p, we did fix a [similar sounding issue](https://github.com/gitkraken/vscode-gitlens/issues/3514) in the pre-release. Can you please see if this issue is still reproducible in the pre-release edition of GitLens?​

You can switch to the pre-release edition of GitLens, by clicking on the "Switch to Pre-Release version of this extension" from the Extensions view.

![image](https://user-images.githubusercontent.com/27739784/240437025-f9f6b0cf-cda3-4b53-973b-3689f1e4481a.png)

**Reply by jq001p** [reply]
That did solve the issue but I did notice that the delete option would be missing from the drop down. If I started in that branch then switched off it in order to delete it the option would not be there until I restarted Visual Studio.
